.chloggen/v1beta1-remove-webhook.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
+
+# The name of the component, or a single word describing the area of concern, (e.g. collector, target allocator, auto-instrumentation, opamp, github action)
+component: collector
+
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Remove collector v1alpha1 defaulting and validating webhooks.
+
+# One or more tracking issues related to the change
+issues: [2736]
+
+# (Optional) One or more lines of additional information to render under the primary note.
+# These lines will be padded with 2 spaces and then inserted directly into the document.
+# Use pipe (|) for multiline entries.
+subtext: The functionality was moved to the collector v1beta1 webhooks.
```

Comment:
will users experience any interruptions as a result of this? Do we need them to do anything? Maybe a call out to ensure webhooks are up to date?

Reply:
Everything should be handled as part of the installation. Maybe if they don't use the cert-manager and setup certs by hand

Reply:
Probably not even that bc the webhook endpoint does not change. I can remove the changelog

Reply:
I think we should keep the changelog, but maybe call out this is mostly an internal change and that people should ensure their webhooks are up-to-date